packages/main/src/Toast.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="ui5-toast-root-parent"
```

Comment:
root parent sounds really strange to me :D 

Recheck the class naming please.

Reply:
Do you have a proposal for a better name ?

Reply:
We agreed upon new naming convention.